model-optimizer/install_prerequisites/install_prerequisites.bat

```diff
@@ -67,10 +76,20 @@ IF /I "%1%" EQU "" (
  )
 )
 
-pip3 install --user -r ..\requirements%postfix%.txt
+IF /I "%2%" EQU "venv" (
+    set USE_VENV="true"
+)
+
+IF %USE_VENV% == "true" (
+    python -m venv "%VENV_DIR%"
+    call "%VENV_DIR%\Scripts\activate.bat"
+)
+
+python -m pip install -U pip
+python -m pip install -r "%ROOT_DIR%..\requirements%postfix%.txt"
```

Comment:
It looks that `\` is missed after `%ROOT_DIR%`

Reply:
ROOT_DIR is equal to `%~dp0` and already ends with `\`